### PR TITLE
Add haydenfetch 

### PR DESCRIPTION
**Pull request by asdfugil**
add haydenfetch. Supersedes #611 .

### CODE REVIEW

**Comment by asdfugil** [comment]
this should satisfy all suggestions in #611 .

**Review by TheRealKeto — APPROVED** [review]
Approved.

**Review by brennanleblanc0 — APPROVED** [review]

File: haydenfetch.mk [hunk 1]
```diff
@@ -0,0 +1,37 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS         += haydenfetch
+HAYDENFETCH_VERSION := 1.0-2
+DEB_HAYDENFETCH_V   ?= $(HAYDENFETCH_VERSION)
+
+haydenfetch-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://github.com/asdfugil/haydenfetch/archive/refs/tags/v$(HAYDENFETCH_VERSION).tar.gz
```
Comment (1Conan) [inline haydenfetch.mk:10]: Use GITHUB_ARCHIVE function

**Review by 1Conan — CHANGES_REQUESTED** [review]

**Review by 1Conan — APPROVED** [review]

**Comment by asdfugil** [comment]
- fixed bad substitution error on some jailbreaks due to shebang not being respected
- added help message
- add more haydens

**Comment by asdfugil** [comment]
bruh when did I close this

**Pull request closed without merging**